core/src/main/resources/error/error-classes.json

```
@@ -774,6 +774,12 @@
       "The expected format is ByteString, but was <unsupported> (<class>)."
     ]
   },
+  "INVALID_COLUMN_NAME" : {
+    "message" : [
+      "Column name <columnName> contains invalid character(s). Please use alias to rename it."
```

The message is slightly generic, and doesn't give a clear picture. From the source code, the error is a datasource specific, and raised when the datasource doesn't support a file name that is the same as a column name. Can we rephrase it:
```suggestion
      "The datasource <datasource> cannot save the column <columnName> because its name contains some characters that are not allowed in file paths. Please, use an alias to rename it."
```